### ISSUE TITLE
Ubiquiti LiteBeam M5 / NanoBeam M5

### ISSUE THREAD

**Issue by RubenKelevra** [freifunk-gluon/gluon#1099]
We discovered those two devices which are pretty cheap and have nice buildin antennas for P-t-P (Litebeam) and as AP/Meshpoint (NanoBeam).

Is Gluon support possible for those two?

**Comment by NeoRaider** [comment]
If these are AirMax "XM" devices, the bullet image should work. If they are "XW", LEDE support is missing at the moment.

**Comment by RubenKelevra** [comment]
Thanks!

We got one LiteBeam M5 installed with stock firmware but it's reachable, so I'll try the Bullet firmware, if we got the right hardware.

Currently we're not using any NanoBeams but they look promising as nanostation-replacement, that's why I asked for them too.

Anything we can do if we got the wrong hardware for the Bullet firmware to help the LEDE development?

**Comment by FreifunkUFO** [comment]
its XW. but there is no downgrade-"stock"-firmware to avoid the "new stock firmware bugs" :-(
flashing with freifunk-firmware is working, but you cant save anything.

very crazy discussions at freifunk-forum, so people say its bad to think about openwrt/lede support :-(

https://forum.freifunk.net/t/litebeam-m5-praxistest/12735

**Comment by NeoRaider** [comment]
Hmm yes, the missing flash unlock support, I need to look into that. What flash chip does the LiteBeam use?

**Comment by FreifunkUFO** [comment]
have a look: 

..dmesg 
[    0.000000] Linux version 4.4.50 (buildbot@builds-02.infra.lede-project.org) (gcc version 5.4.0 (LEDE GCC 5.4.0 r3101-bce140e) ) #0 Mon Feb 20 17:13:44 2017
[    0.000000] MyLoader: sysp=8c84edad, boardp=85b59785, parts=a58521a0
[    0.000000] bootconsole [early0] enabled
[    0.000000] CPU0 revision is: 0001974c (MIPS 74Kc)
[    0.000000] SoC: Atheros AR9342 rev 2
[    0.000000] Determined physical RAM map:
[    0.000000]  memory: 04000000 @ 00000000 (usable)
[    0.000000] Initrd not found or empty - disabling initrd
[    0.000000] No valid device tree found, continuing without
[    0.000000] Zone ranges:
[    0.000000]   Normal   [mem 0x0000000000000000-0x0000000003ffffff]
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x0000000000000000-0x0000000003ffffff]
[    0.000000] Initmem setup node 0 [mem 0x0000000000000000-0x0000000003ffffff]
[    0.000000] On node 0 totalpages: 16384
[    0.000000] free_area_init_node: node 0, pgdat 803d04d0, node_mem_map 81000000
[    0.000000]   Normal zone: 128 pages used for memmap
[    0.000000]   Normal zone: 0 pages reserved
[    0.000000]   Normal zone: 16384 pages, LIFO batch:3
[    0.000000] Primary instruction cache 64kB, VIPT, 4-way, linesize 32 bytes.
[    0.000000] Primary data cache 32kB, 4-way, VIPT, cache aliases, linesize 32 bytes
[    0.000000] pcpu-alloc: s0 r0 d32768 u32768 alloc=1*32768
[    0.000000] pcpu-alloc: [0] 0 
[    0.000000] Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 16256
[    0.000000] Kernel command line:  board=UBNT-LOCO-XW mtdparts=spi0.0:256k(u-boot)ro,64k(u-boot-env)ro,7552k(firmware),256k(cfg)ro,64k(EEPROM)ro console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd
[    0.000000] PID hash table entries: 256 (order: -2, 1024 bytes)
[    0.000000] Dentry cache hash table entries: 8192 (order: 3, 32768 bytes)
[    0.000000] Inode-cache hash table entries: 4096 (order: 2, 16384 bytes)
[    0.000000] Writing ErrCtl register=00000000
[    0.000000] Readback ErrCtl register=00000000
[    0.000000] Memory: 60352K/65536K available (3071K kernel code, 165K rwdata, 412K rodata, 312K init, 205K bss, 5184K reserved, 0K cma-reserved)
[    0.000000] SLUB: HWalign=32, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
[    0.000000] NR_IRQS:51
[    0.000000] Clocks: CPU:535.000MHz, DDR:400.000MHz, AHB:200.000MHz, Ref:40.000MHz
[    0.000000] clocksource: MIPS: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7144898866 ns
[    0.000010] sched_clock: 32 bits at 267MHz, resolution 3ns, wraps every 8027976190ns
[    0.008312] Calibrating delay loop... 266.64 BogoMIPS (lpj=1333248)
[    0.091155] pid_max: default: 32768 minimum: 301
[    0.096214] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes)
[    0.103270] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes)
[    0.113601] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
[    0.125386] NET: Registered protocol family 16
[    0.131661] MIPS: machine is Ubiquiti Loco M XW
[    0.367902] clocksource: Switched to clocksource MIPS
[    0.374557] NET: Registered protocol family 2
[    0.380300] TCP established hash table entries: 1024 (order: 0, 4096 bytes)
[    0.387736] TCP bind hash table entries: 1024 (order: 0, 4096 bytes)
[    0.394569] TCP: Hash tables configured (established 1024 bind 1024)
[    0.401454] UDP hash table entries: 256 (order: 0, 4096 bytes)
[    0.407689] UDP-Lite hash table entries: 256 (order: 0, 4096 bytes)
[    0.414704] NET: Registered protocol family 1
[    0.419423] PCI: CLS 0 bytes, default 32
[    0.424247] futex hash table entries: 256 (order: -1, 3072 bytes)
[    0.430881] Crashlog allocated RAM at address 0x3f00000
[    0.453157] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    0.459431] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
[    0.472778] io scheduler noop registered
[    0.476953] io scheduler deadline registered (default)
[    0.482742] Serial: 8250/16550 driver, 16 ports, IRQ sharing enabled
[    0.492674] console [ttyS0] disabled
[    0.516596] serial8250.0: ttyS0 at MMIO 0x18020000 (irq = 11, base_baud = 2500000) is a 16550A
[    0.525793] console [ttyS0] enabled
[    0.533217] bootconsole [early0] disabled
[    0.547394] m25p80 spi0.0: found mx25l6405d, expected m25p80
[    0.553239] m25p80 spi0.0: mx25l6405d (8192 Kbytes)
[    0.558230] 5 cmdlinepart partitions found on MTD device spi0.0
[    0.564233] Creating 5 MTD partitions on "spi0.0":
[    0.569112] 0x000000000000-0x000000040000 : "u-boot"
[    0.576687] 0x000000040000-0x000000050000 : "u-boot-env"
[    0.584041] 0x000000050000-0x0000007b0000 : "firmware"
[    0.600982] 2 uimage-fw partitions found on MTD device firmware
[    0.607012] 0x000000050000-0x000000190000 : "kernel"
[    0.613689] 0x000000190000-0x0000007b0000 : "rootfs"
[    0.620677] mtd: device 4 (rootfs) set to be root filesystem
[    0.626463] 1 squashfs-split partitions found on MTD device rootfs
[    0.632807] 0x0000003c0000-0x0000007b0000 : "rootfs_data"
[    0.640249] 0x0000007b0000-0x0000007f0000 : "cfg"
[    0.646917] 0x0000007f0000-0x000000800000 : "EEPROM"
[    0.660500] libphy: ag71xx_mdio: probed
[    1.328416] ag71xx ag71xx.0: connected to PHY at ag71xx-mdio.0:01 [uid=004dd023, driver=Atheros 8032 ethernet]
[    1.339370] eth0: Atheros AG71xx at 0xb9000000, irq 4, mode:MII
[    1.347597] NET: Registered protocol family 10
[    1.356789] NET: Registered protocol family 17
[    1.361461] bridge: automatic filtering via arp/ip/ip6tables has been deprecated. Update your scripts to load br_netfilter if you need this.
[    1.374396] 8021q: 802.1Q VLAN Support v1.8
[    1.385283] VFS: Mounted root (squashfs filesystem) readonly on device 31:4.
[    1.394581] Freeing unused kernel memory: 312K (803f2000 - 80440000)
[    2.612455] init: Console is alive
[    2.616155] init: - watchdog -
[    3.680679] usbcore: registered new interface driver usbfs
[    3.686364] usbcore: registered new interface driver hub
[    3.691944] usbcore: registered new device driver usb
[    3.703215] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    3.711581] ehci-platform: EHCI generic platform driver
[    3.720047] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
[    3.727770] ohci-platform: OHCI generic platform driver
[    3.736987] init: - preinit -
[    4.369681] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
[    4.398750] random: procd: uninitialized urandom read (4 bytes read, 7 bits of entropy available)
[    6.429651] eth0: link up (100Mbps/Full duplex)
[    6.434300] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[    7.545899] mount_root: jffs2 not ready yet, using temporary tmpfs overlay
[    7.557440] urandom-seed: Seed file not found (/etc/urandom.seed)
[    7.779659] eth0: link down
[    7.795163] procd: - early -
[    7.798297] procd: - watchdog -
[    8.447868] procd: - ubus -
[    8.560598] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.570774] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.580529] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.589741] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.599366] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.608572] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.617993] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.627336] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
[    8.636962] procd: - init -
[    9.063307] ip6_tables: (C) 2000-2006 Netfilter Core Team
[    9.081790] Loading modules backported from Linux version wt-2016-10-03-1-g6fcb1a6
[    9.089548] Backport generated by backports.git backports-20160324-9-g0e38f5c
[    9.100453] ip_tables: (C) 2000-2006 Netfilter Core Team
[    9.115557] nf_conntrack version 0.5.0 (947 buckets, 3788 max)
[    9.159942] xt_time: kernel timezone is -0000
[    9.232523] PPP generic driver version 2.4.2
[    9.239829] NET: Registered protocol family 24
[    9.292423] ath: EEPROM regdomain: 0x0
[    9.292448] ath: EEPROM indicates default country code should be used
[    9.292458] ath: doing EEPROM country->regdmn map search
[    9.292481] ath: country maps to regdmn code: 0x3a
[    9.292494] ath: Country alpha2 being used: US
[    9.292504] ath: Regpair used: 0x3a
[    9.305379] ieee80211 phy0: Selected rate control algorithm 'minstrel_ht'
[    9.311581] ieee80211 phy0: Atheros AR9340 Rev:2 mem=0xb8100000, irq=47
[   10.694533] random: jshn: uninitialized urandom read (4 bytes read, 17 bits of entropy available)
[   18.068323] jffs2_scan_eraseblock(): End of filesystem marker found at 0x0
[   18.075323] jffs2_build_filesystem(): unlocking the mtd device... done.
[   18.082101] jffs2_build_filesystem(): erasing all blocks after the end marker... 
[   18.909690] jffs2: Newly-erased block contained word 0xdeadc0de at offset 0x00340000
[   18.939588] jffs2: Newly-erased block contained word 0xdeadc0de at offset 0x00330000
[   18.969603] jffs2: Newly-erased block contained word 0x250205f6 at offset 0x00320000
[   18.999579] jffs2: Newly-erased block contained word 0xbda99bd5 at offset 0x00310000
[   19.029619] jffs2: Newly-erased block contained word 0x6865fce4 at offset 0x00300000
[   19.059572] jffs2: Newly-erased block contained word 0xe3a608d5 at offset 0x002f0000
[   19.089618] jffs2: Newly-erased block contained word 0x2d375190 at offset 0x002e0000
[   19.119583] jffs2: Newly-erased block contained word 0xc0fe6c02 at offset 0x002d0000
[   19.149613] jffs2: Newly-erased block contained word 0xc1df901c at offset 0x002c0000
[   19.179542] jffs2: Newly-erased block contained word 0x7b13c031 at offset 0x002b0000
[   19.209614] jffs2: Newly-erased block contained word 0x82b04523 at offset 0x002a0000
[   19.219742] jffs2: Newly-erased block contained word 0x700eb9e1 at offset 0x00290000
[   19.249570] jffs2: Newly-erased block contained word 0xe79043f7 at offset 0x00280000
[   19.279578] jffs2: Newly-erased block contained word 0x3151c923 at offset 0x00270000
[   19.309612] jffs2: Newly-erased block contained word 0xcb39c041 at offset 0x00260000
[   19.339543] jffs2: Newly-erased block contained word 0xa07230f8 at offset 0x00250000
[   19.369609] jffs2: Newly-erased block contained word 0x655fb5e2 at offset 0x00240000
[   19.399585] jffs2: Newly-erased block contained word 0x6eaf6479 at offset 0x00230000
[   19.429611] jffs2: Newly-erased block contained word 0xbb8bafa4 at offset 0x00220000
[   19.459540] jffs2: Newly-erased block contained word 0x5025d8ae at offset 0x00210000
[   19.489611] jffs2: Newly-erased block contained word 0xa78670c4 at offset 0x00200000
[   19.519589] jffs2: Newly-erased block contained word 0x242adac at offset 0x001f0000
[   19.549616] jffs2: Newly-erased block contained word 0x3f534d7b at offset 0x001e0000
[   19.579541] jffs2: Newly-erased block contained word 0x31836246 at offset 0x001d0000
[   19.609609] jffs2: Newly-erased block contained word 0x25dd0105 at offset 0x001c0000
[   19.639586] jffs2: Newly-erased block contained word 0x508ad090 at offset 0x001b0000
[   19.669621] jffs2: Newly-erased block contained word 0xaba29ec7 at offset 0x001a0000
[   19.699541] jffs2: Newly-erased block contained word 0x4943aa17 at offset 0x00190000
[   19.729620] jffs2: Newly-erased block contained word 0xf3fb6441 at offset 0x00180000
[   19.749554] jffs2: Newly-erased block contained word 0x44c82470 at offset 0x00170000
[   19.769631] jffs2: Newly-erased block contained word 0x91334203 at offset 0x00160000
[   19.799577] jffs2: Newly-erased block contained word 0xa9f93a4a at offset 0x00150000
[   19.829609] jffs2: Newly-erased block contained word 0x4ee085f5 at offset 0x00140000
[   19.859578] jffs2: Newly-erased block contained word 0x1396075 at offset 0x00130000
[   19.889611] jffs2: Newly-erased block contained word 0xd9726016 at offset 0x00120000
[   19.919540] jffs2: Newly-erased block contained word 0x8b98684b at offset 0x00110000
[   19.949606] jffs2: Newly-erased block contained word 0xa07e772d at offset 0x00100000
[   19.979576] jffs2: Newly-erased block contained word 0xdd223c98 at offset 0x000f0000
[   20.009609] jffs2: Newly-erased block contained word 0x29725f8c at offset 0x000e0000
[   20.039546] jffs2: Newly-erased block contained word 0x7ecaa964 at offset 0x000d0000
[   20.069617] jffs2: Newly-erased block contained word 0x7ebef884 at offset 0x000c0000
[   20.099578] jffs2: Newly-erased block contained word 0x460ab758 at offset 0x000b0000
[   20.129607] jffs2: Newly-erased block contained word 0xdf706cd9 at offset 0x000a0000
[   20.159540] jffs2: Newly-erased block contained word 0xacdac38f at offset 0x00090000
[   20.189610] jffs2: Newly-erased block contained word 0x69aba5b5 at offset 0x00080000
[   20.219581] jffs2: Newly-erased block contained word 0x799b2cac at offset 0x00070000
[   20.249573] jffs2: Newly-erased block contained word 0xc70fe986 at offset 0x00060000
[   20.269571] jffs2: Newly-erased block contained word 0x7d49e374 at offset 0x00050000
[   20.289625] jffs2: Newly-erased block contained word 0xfec8a523 at offset 0x00040000
[   20.319577] jffs2: Newly-erased block contained word 0x9bf1c220 at offset 0x00030000
[   20.349616] jffs2: Newly-erased block contained word 0x4c62fad1 at offset 0x00020000
[   20.379577] jffs2: Newly-erased block contained word 0x33cd8359 at offset 0x00010000
[   20.409572] jffs2: Newly-erased block contained word 0xdeadc0de at offset 0x00000000
[   20.417441] done.
[   20.419465] jffs2: notice: (900) jffs2_build_xattr_subsystem: complete building xattr subsystem, 0 of xdatum (0 unchecked, 0 orphan) and 0 of xref (0 dead, 0 orphan) found.
[   22.182117] device eth0 entered promiscuous mode
[   22.219749] IPv6: ADDRCONF(NETDEV_UP): br-lan: link is not ready
[   24.549711] eth0: link up (100Mbps/Full duplex)
[   24.554383] br-lan: port 1(eth0) entered forwarding state
[   24.559952] br-lan: port 1(eth0) entered forwarding state
[   24.588024] IPv6: ADDRCONF(NETDEV_CHANGE): br-lan: link becomes ready
[   26.557942] br-lan: port 1(eth0) entered forwarding state
[   75.327939] random: nonblocking pool is initialized
root@LEDE:~# cat /proc/mtd 
dev:    size   erasesize  name
mtd0: 00040000 00010000 "u-boot"
mtd1: 00010000 00010000 "u-boot-env"
mtd2: 00760000 00010000 "firmware"
mtd3: 00140000 00010000 "kernel"
mtd4: 00620000 00010000 "rootfs"
mtd5: 003f0000 00010000 "rootfs_data"
mtd6: 00040000 00010000 "cfg"
mtd7: 00010000 00010000 "EEPROM"
root@LEDE:~# cat /tmp/sysinfo/*
loco-m-xw
Ubiquiti Loco M XW
root@LEDE:~# uname -a
Linux LEDE 4.4.50 #0 Mon Feb 20 17:13:44 2017 mips GNU/Linux
root@LEDE:~# cat /etc/openwrt_*
DISTRIB_ID='LEDE'
DISTRIB_RELEASE='17.01.0'
DISTRIB_REVISION='r3205-59508e3'
DISTRIB_CODENAME='reboot'
DISTRIB_TARGET='ar71xx/generic'
DISTRIB_ARCH='mips_24kc'
DISTRIB_DESCRIPTION='LEDE Reboot 17.01.0 r3205-59508e3'
DISTRIB_TAINTS='no-all'
r3205-59508e3
root@LEDE:~#

**Comment by FreifunkUFO** [comment]
logread 


Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.000000] Linux version 4.4.50 (buildbot@builds-02.infra.lede-project.org) (gcc version 5.4.0 (LEDE GCC 5.4.0 r3101-bce140e) ) #0 Mon Feb 20 17:13:44 2017
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000] MyLoader: sysp=8c84edad, boardp=85b59785, parts=a58521a0
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] bootconsole [early0] enabled
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] CPU0 revision is: 0001974c (MIPS 74Kc)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] SoC: Atheros AR9342 rev 2
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Determined physical RAM map:
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000]  memory: 04000000 @ 00000000 (usable)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Initrd not found or empty - disabling initrd
Mon Feb 20 17:13:48 2017 kern.warn kernel: [    0.000000] No valid device tree found, continuing without
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Zone ranges:
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000]   Normal   [mem 0x0000000000000000-0x0000000003ffffff]
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Movable zone start for each node
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Early memory node ranges
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000]   node   0: [mem 0x0000000000000000-0x0000000003ffffff]
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Initmem setup node 0 [mem 0x0000000000000000-0x0000000003ffffff]
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000] On node 0 totalpages: 16384
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000] free_area_init_node: node 0, pgdat 803d04d0, node_mem_map 81000000
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000]   Normal zone: 128 pages used for memmap
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000]   Normal zone: 0 pages reserved
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000]   Normal zone: 16384 pages, LIFO batch:3
Mon Feb 20 17:13:48 2017 kern.warn kernel: [    0.000000] Primary instruction cache 64kB, VIPT, 4-way, linesize 32 bytes.
Mon Feb 20 17:13:48 2017 kern.warn kernel: [    0.000000] Primary data cache 32kB, 4-way, VIPT, cache aliases, linesize 32 bytes
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000] pcpu-alloc: s0 r0 d32768 u32768 alloc=1*32768
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.000000] pcpu-alloc: [0] 0 
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Built 1 zonelists in Zone order, mobility grouping on.  Total pages: 16256
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.000000] Kernel command line:  board=UBNT-LOCO-XW mtdparts=spi0.0:256k(u-boot)ro,64k(u-boot-env)ro,7552k(firmware),256k(cfg)ro,64k(EEPROM)ro console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] PID hash table entries: 256 (order: -2, 1024 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Dentry cache hash table entries: 8192 (order: 3, 32768 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Inode-cache hash table entries: 4096 (order: 2, 16384 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Writing ErrCtl register=00000000
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Readback ErrCtl register=00000000
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Memory: 60352K/65536K available (3071K kernel code, 165K rwdata, 412K rodata, 312K init, 205K bss, 5184K reserved, 0K cma-reserved)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] SLUB: HWalign=32, Order=0-3, MinObjects=0, CPUs=1, Nodes=1
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] NR_IRQS:51
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] Clocks: CPU:535.000MHz, DDR:400.000MHz, AHB:200.000MHz, Ref:40.000MHz
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000000] clocksource: MIPS: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7144898866 ns
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.000010] sched_clock: 32 bits at 267MHz, resolution 3ns, wraps every 8027976190ns
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.008312] Calibrating delay loop... 266.64 BogoMIPS (lpj=1333248)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.091155] pid_max: default: 32768 minimum: 301
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.096214] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.103270] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.113601] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.125386] NET: Registered protocol family 16
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.131661] MIPS: machine is Ubiquiti Loco M XW
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.367902] clocksource: Switched to clocksource MIPS
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.374557] NET: Registered protocol family 2
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.380300] TCP established hash table entries: 1024 (order: 0, 4096 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.387736] TCP bind hash table entries: 1024 (order: 0, 4096 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.394569] TCP: Hash tables configured (established 1024 bind 1024)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.401454] UDP hash table entries: 256 (order: 0, 4096 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.407689] UDP-Lite hash table entries: 256 (order: 0, 4096 bytes)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.414704] NET: Registered protocol family 1
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    0.419423] PCI: CLS 0 bytes, default 32
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.424247] futex hash table entries: 256 (order: -1, 3072 bytes)
Mon Feb 20 17:13:48 2017 kern.warn kernel: [    0.430881] Crashlog allocated RAM at address 0x3f00000
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.453157] squashfs: version 4.0 (2009/01/31) Phillip Lougher
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.459431] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.472778] io scheduler noop registered
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.476953] io scheduler deadline registered (default)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.482742] Serial: 8250/16550 driver, 16 ports, IRQ sharing enabled
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.492674] console [ttyS0] disabled
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.516596] serial8250.0: ttyS0 at MMIO 0x18020000 (irq = 11, base_baud = 2500000) is a 16550A
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.525793] console [ttyS0] enabled
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.533217] bootconsole [early0] disabled
Mon Feb 20 17:13:48 2017 kern.warn kernel: [    0.547394] m25p80 spi0.0: found mx25l6405d, expected m25p80
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.553239] m25p80 spi0.0: mx25l6405d (8192 Kbytes)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.558230] 5 cmdlinepart partitions found on MTD device spi0.0
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.564233] Creating 5 MTD partitions on "spi0.0":
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.569112] 0x000000000000-0x000000040000 : "u-boot"
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.576687] 0x000000040000-0x000000050000 : "u-boot-env"
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.584041] 0x000000050000-0x0000007b0000 : "firmware"
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.600982] 2 uimage-fw partitions found on MTD device firmware
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.607012] 0x000000050000-0x000000190000 : "kernel"
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.613689] 0x000000190000-0x0000007b0000 : "rootfs"
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.620677] mtd: device 4 (rootfs) set to be root filesystem
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.626463] 1 squashfs-split partitions found on MTD device rootfs
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.632807] 0x0000003c0000-0x0000007b0000 : "rootfs_data"
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.640249] 0x0000007b0000-0x0000007f0000 : "cfg"
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    0.646917] 0x0000007f0000-0x000000800000 : "EEPROM"
Mon Feb 20 17:13:48 2017 kern.info kernel: [    0.660500] libphy: ag71xx_mdio: probed
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.328416] ag71xx ag71xx.0: connected to PHY at ag71xx-mdio.0:01 [uid=004dd023, driver=Atheros 8032 ethernet]
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.339370] eth0: Atheros AG71xx at 0xb9000000, irq 4, mode:MII
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.347597] NET: Registered protocol family 10
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.356789] NET: Registered protocol family 17
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.361461] bridge: automatic filtering via arp/ip/ip6tables has been deprecated. Update your scripts to load br_netfilter if you need this.
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.374396] 8021q: 802.1Q VLAN Support v1.8
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.385283] VFS: Mounted root (squashfs filesystem) readonly on device 31:4.
Mon Feb 20 17:13:48 2017 kern.info kernel: [    1.394581] Freeing unused kernel memory: 312K (803f2000 - 80440000)
Mon Feb 20 17:13:48 2017 user.info kernel: [    2.612455] init: Console is alive
Mon Feb 20 17:13:48 2017 user.info kernel: [    2.616155] init: - watchdog -
Mon Feb 20 17:13:48 2017 kern.info kernel: [    3.680679] usbcore: registered new interface driver usbfs
Mon Feb 20 17:13:48 2017 kern.info kernel: [    3.686364] usbcore: registered new interface driver hub
Mon Feb 20 17:13:48 2017 kern.info kernel: [    3.691944] usbcore: registered new device driver usb
Mon Feb 20 17:13:48 2017 kern.info kernel: [    3.703215] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
Mon Feb 20 17:13:48 2017 kern.info kernel: [    3.711581] ehci-platform: EHCI generic platform driver
Mon Feb 20 17:13:48 2017 kern.info kernel: [    3.720047] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
Mon Feb 20 17:13:48 2017 kern.info kernel: [    3.727770] ohci-platform: OHCI generic platform driver
Mon Feb 20 17:13:48 2017 user.info kernel: [    3.736987] init: - preinit -
Mon Feb 20 17:13:48 2017 kern.info kernel: [    4.369681] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    4.398750] random: procd: uninitialized urandom read (4 bytes read, 7 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    6.429651] eth0: link up (100Mbps/Full duplex)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    6.434300] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
Mon Feb 20 17:13:48 2017 user.notice kernel: [    7.545899] mount_root: jffs2 not ready yet, using temporary tmpfs overlay
Mon Feb 20 17:13:48 2017 user.warn kernel: [    7.557440] urandom-seed: Seed file not found (/etc/urandom.seed)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    7.779659] eth0: link down
Mon Feb 20 17:13:48 2017 user.info kernel: [    7.795163] procd: - early -
Mon Feb 20 17:13:48 2017 user.info kernel: [    7.798297] procd: - watchdog -
Mon Feb 20 17:13:48 2017 user.info kernel: [    8.447868] procd: - ubus -
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.560598] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.570774] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.580529] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.589741] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.599366] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.608572] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.617993] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 kern.notice kernel: [    8.627336] random: ubusd: uninitialized urandom read (4 bytes read, 14 bits of entropy available)
Mon Feb 20 17:13:48 2017 user.info kernel: [    8.636962] procd: - init -
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.063307] ip6_tables: (C) 2000-2006 Netfilter Core Team
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.081790] Loading modules backported from Linux version wt-2016-10-03-1-g6fcb1a6
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.089548] Backport generated by backports.git backports-20160324-9-g0e38f5c
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.100453] ip_tables: (C) 2000-2006 Netfilter Core Team
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.115557] nf_conntrack version 0.5.0 (947 buckets, 3788 max)
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.159942] xt_time: kernel timezone is -0000
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.232523] PPP generic driver version 2.4.2
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.239829] NET: Registered protocol family 24
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    9.292423] ath: EEPROM regdomain: 0x0
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    9.292448] ath: EEPROM indicates default country code should be used
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    9.292458] ath: doing EEPROM country->regdmn map search
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    9.292481] ath: country maps to regdmn code: 0x3a
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    9.292494] ath: Country alpha2 being used: US
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    9.292504] ath: Regpair used: 0x3a
Mon Feb 20 17:13:48 2017 kern.debug kernel: [    9.305379] ieee80211 phy0: Selected rate control algorithm 'minstrel_ht'
Mon Feb 20 17:13:48 2017 kern.info kernel: [    9.311581] ieee80211 phy0: Atheros AR9340 Rev:2 mem=0xb8100000, irq=47
Mon Feb 20 17:13:48 2017 kern.notice kernel: [   10.694533] random: jshn: uninitialized urandom read (4 bytes read, 17 bits of entropy available)
Mon Feb 20 17:13:50 2017 user.notice : Added device handler type: 8021ad
Mon Feb 20 17:13:50 2017 user.notice : Added device handler type: 8021q
Mon Feb 20 17:13:50 2017 user.notice : Added device handler type: macvlan
Mon Feb 20 17:13:50 2017 user.notice : Added device handler type: bridge
Mon Feb 20 17:13:50 2017 user.notice : Added device handler type: Network device
Mon Feb 20 17:13:50 2017 user.notice : Added device handler type: tunnel
Mon Feb 20 17:13:50 2017 user.notice : 'radio0' is disabled
Mon Feb 20 17:13:51 2017 user.notice : 'radio0' is disabled
Mon Feb 20 17:13:53 2017 daemon.err mount_root: no jffs2 marker found
Mon Feb 20 17:13:53 2017 kern.warn kernel: [   18.068323] jffs2_scan_eraseblock(): End of filesystem marker found at 0x0
Mon Feb 20 17:13:53 2017 kern.warn kernel: [   18.075323] jffs2_build_filesystem(): unlocking the mtd device... done.
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   18.082101] jffs2_build_filesystem(): erasing all blocks after the end marker... 
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   18.909690] jffs2: Newly-erased block contained word 0xdeadc0de at offset 0x00340000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   18.939588] jffs2: Newly-erased block contained word 0xdeadc0de at offset 0x00330000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   18.969603] jffs2: Newly-erased block contained word 0x250205f6 at offset 0x00320000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   18.999579] jffs2: Newly-erased block contained word 0xbda99bd5 at offset 0x00310000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.029619] jffs2: Newly-erased block contained word 0x6865fce4 at offset 0x00300000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.059572] jffs2: Newly-erased block contained word 0xe3a608d5 at offset 0x002f0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.089618] jffs2: Newly-erased block contained word 0x2d375190 at offset 0x002e0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.119583] jffs2: Newly-erased block contained word 0xc0fe6c02 at offset 0x002d0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.149613] jffs2: Newly-erased block contained word 0xc1df901c at offset 0x002c0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.179542] jffs2: Newly-erased block contained word 0x7b13c031 at offset 0x002b0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.209614] jffs2: Newly-erased block contained word 0x82b04523 at offset 0x002a0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.219742] jffs2: Newly-erased block contained word 0x700eb9e1 at offset 0x00290000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.249570] jffs2: Newly-erased block contained word 0xe79043f7 at offset 0x00280000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.279578] jffs2: Newly-erased block contained word 0x3151c923 at offset 0x00270000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.309612] jffs2: Newly-erased block contained word 0xcb39c041 at offset 0x00260000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.339543] jffs2: Newly-erased block contained word 0xa07230f8 at offset 0x00250000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.369609] jffs2: Newly-erased block contained word 0x655fb5e2 at offset 0x00240000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.399585] jffs2: Newly-erased block contained word 0x6eaf6479 at offset 0x00230000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.429611] jffs2: Newly-erased block contained word 0xbb8bafa4 at offset 0x00220000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.459540] jffs2: Newly-erased block contained word 0x5025d8ae at offset 0x00210000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.489611] jffs2: Newly-erased block contained word 0xa78670c4 at offset 0x00200000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.519589] jffs2: Newly-erased block contained word 0x242adac at offset 0x001f0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.549616] jffs2: Newly-erased block contained word 0x3f534d7b at offset 0x001e0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.579541] jffs2: Newly-erased block contained word 0x31836246 at offset 0x001d0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.609609] jffs2: Newly-erased block contained word 0x25dd0105 at offset 0x001c0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.639586] jffs2: Newly-erased block contained word 0x508ad090 at offset 0x001b0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.669621] jffs2: Newly-erased block contained word 0xaba29ec7 at offset 0x001a0000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.699541] jffs2: Newly-erased block contained word 0x4943aa17 at offset 0x00190000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.729620] jffs2: Newly-erased block contained word 0xf3fb6441 at offset 0x00180000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.749554] jffs2: Newly-erased block contained word 0x44c82470 at offset 0x00170000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.769631] jffs2: Newly-erased block contained word 0x91334203 at offset 0x00160000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.799577] jffs2: Newly-erased block contained word 0xa9f93a4a at offset 0x00150000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.829609] jffs2: Newly-erased block contained word 0x4ee085f5 at offset 0x00140000
Mon Feb 20 17:13:54 2017 kern.warn kernel: [   19.859578] jffs2: Newly-erased block contained word 0x1396075 at offset 0x00130000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   19.889611] jffs2: Newly-erased block contained word 0xd9726016 at offset 0x00120000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   19.919540] jffs2: Newly-erased block contained word 0x8b98684b at offset 0x00110000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   19.949606] jffs2: Newly-erased block contained word 0xa07e772d at offset 0x00100000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   19.979576] jffs2: Newly-erased block contained word 0xdd223c98 at offset 0x000f0000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.009609] jffs2: Newly-erased block contained word 0x29725f8c at offset 0x000e0000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.039546] jffs2: Newly-erased block contained word 0x7ecaa964 at offset 0x000d0000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.069617] jffs2: Newly-erased block contained word 0x7ebef884 at offset 0x000c0000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.099578] jffs2: Newly-erased block contained word 0x460ab758 at offset 0x000b0000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.129607] jffs2: Newly-erased block contained word 0xdf706cd9 at offset 0x000a0000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.159540] jffs2: Newly-erased block contained word 0xacdac38f at offset 0x00090000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.189610] jffs2: Newly-erased block contained word 0x69aba5b5 at offset 0x00080000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.219581] jffs2: Newly-erased block contained word 0x799b2cac at offset 0x00070000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.249573] jffs2: Newly-erased block contained word 0xc70fe986 at offset 0x00060000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.269571] jffs2: Newly-erased block contained word 0x7d49e374 at offset 0x00050000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.289625] jffs2: Newly-erased block contained word 0xfec8a523 at offset 0x00040000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.319577] jffs2: Newly-erased block contained word 0x9bf1c220 at offset 0x00030000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.349616] jffs2: Newly-erased block contained word 0x4c62fad1 at offset 0x00020000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.379577] jffs2: Newly-erased block contained word 0x33cd8359 at offset 0x00010000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.409572] jffs2: Newly-erased block contained word 0xdeadc0de at offset 0x00000000
Mon Feb 20 17:13:55 2017 kern.warn kernel: [   20.417441] done.
Mon Feb 20 17:13:55 2017 kern.notice kernel: [   20.419465] jffs2: notice: (900) jffs2_build_xattr_subsystem: complete building xattr subsystem, 0 of xdatum (0 unchecked, 0 orphan) and 0 of xref (0 dead, 0 orphan) found.
Mon Feb 20 17:13:55 2017 daemon.info mount_root: performing overlay whiteout
Mon Feb 20 17:13:57 2017 kern.info kernel: [   22.182117] device eth0 entered promiscuous mode
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Interface 'lan' is enabled
Mon Feb 20 17:13:57 2017 kern.info kernel: [   22.219749] IPv6: ADDRCONF(NETDEV_UP): br-lan: link is not ready
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Interface 'lan' is setting up now
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Interface 'lan' is now up
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Interface 'loopback' is enabled
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Interface 'loopback' is setting up now
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Interface 'loopback' is now up
Mon Feb 20 17:13:57 2017 daemon.info odhcpd[850]: Raising SIGUSR1 due to address change on br-lan
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Network device 'lo' link is up
Mon Feb 20 17:13:57 2017 daemon.notice netifd: Interface 'loopback' has link connectivity 
Mon Feb 20 17:13:58 2017 user.notice firewall: Reloading firewall due to ifup of lan (br-lan)
Mon Feb 20 17:13:58 2017 daemon.info odhcpd[850]: Initial RA router lifetime 0, 1 address(es) available on br-lan
Mon Feb 20 17:13:58 2017 daemon.notice odhcpd[850]: Failed to send to ff02::1%br-lan (Address not available)
Mon Feb 20 17:13:59 2017 daemon.info procd: - init complete -
Mon Feb 20 17:13:59 2017 kern.info kernel: [   24.549711] eth0: link up (100Mbps/Full duplex)
Mon Feb 20 17:13:59 2017 kern.info kernel: [   24.554383] br-lan: port 1(eth0) entered forwarding state
Mon Feb 20 17:13:59 2017 kern.info kernel: [   24.559952] br-lan: port 1(eth0) entered forwarding state
Mon Feb 20 17:13:59 2017 daemon.notice netifd: Network device 'eth0' link is up
Mon Feb 20 17:13:59 2017 daemon.notice netifd: bridge 'br-lan' link is up
Mon Feb 20 17:13:59 2017 daemon.notice netifd: Interface 'lan' has link connectivity 
Mon Feb 20 17:13:59 2017 kern.info kernel: [   24.588024] IPv6: ADDRCONF(NETDEV_CHANGE): br-lan: link becomes ready
Mon Feb 20 17:14:00 2017 daemon.info odhcpd[850]: Initial RA router lifetime 0, 1 address(es) available on br-lan
Mon Feb 20 17:14:00 2017 daemon.notice odhcpd[850]: Failed to send to ff02::1%br-lan (Address not available)
Mon Feb 20 17:14:01 2017 kern.info kernel: [   26.557942] br-lan: port 1(eth0) entered forwarding state
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq[1070]: started, version 2.76 cachesize 150
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq[1070]: compile time options: IPv6 GNU-getopt no-DBus no-i18n no-IDN DHCP no-DHCPv6 no-Lua TFTP no-conntrack no-ipset no-auth no-DNSSEC no-ID loop-detect inotify
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq[1070]: DNS service limited to local subnets
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq-dhcp[1070]: DHCP, IP range 192.168.1.100 -- 192.168.1.249, lease time 12h
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq[1070]: using local addresses only for domain lan
Mon Feb 20 17:14:04 2017 daemon.warn dnsmasq[1070]: no servers found in /tmp/resolv.conf.auto, will retry
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq[1070]: read /etc/hosts - 4 addresses
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq[1070]: read /tmp/hosts/odhcpd - 0 addresses
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq[1070]: read /tmp/hosts/dhcp.cfg02411c - 2 addresses
Mon Feb 20 17:14:04 2017 daemon.info dnsmasq-dhcp[1070]: read /etc/ethers - 0 addresses
Mon Feb 20 17:14:07 2017 authpriv.info dropbear[1153]: Not backgrounding
Mon Feb 20 17:14:18 2017 authpriv.info dropbear[1156]: Child connection from fe80::c66e:1fff:fefe:d886%br-lan:54132
Mon Feb 20 17:14:24 2017 authpriv.notice dropbear[1156]: Auth succeeded with blank password for 'root' from fe80::c66e:1fff:fefe:d886%br-lan:54132
Mon Feb 20 17:14:50 2017 kern.notice kernel: [   75.327939] random: nonblocking pool is initialized
Mon Feb 20 17:14:50 2017 daemon.info urandom_seed[1009]: Seed saved (/etc/urandom.seed)
root@LEDE:~#

**Comment by NeoRaider** [comment]
Weird, unlocking of Macronix flash *should* work. There must still be another issue in the flash driver...

**Comment by FreifunkUFO** [comment]
i was repeating the tests, but no luck:

- tftp ubnt 5.6.15 and then via webui lede-17.01.1

- tftp ubnt 5.6.15, via webui ubnt 6.0.3, then via webui   lede 17.01.1

**Comment by FreifunkUFO** [comment]
but now mtd is slightly different (rootfs-data)

root@LEDE:~# cat /proc/mtd 
dev:    size   erasesize  name
mtd0: 00040000 00010000 "u-boot"
mtd1: 00010000 00010000 "u-boot-env"
mtd2: 00760000 00010000 "firmware"
mtd3: 00140000 00010000 "kernel"
mtd4: 00620000 00010000 "rootfs"
mtd5: 003e0000 00010000 "rootfs_data"
mtd6: 00040000 00010000 "cfg"
mtd7: 00010000 00010000 "EEPROM"
root@LEDE:~#

**Comment by RubenKelevra** [comment]
I'm watching you at work looks promising. Hope @neoraider can find the missing piece.

What's up with the NanoBeam M5? Is this the same hardware or might it be worth testing in parallel?


Best regards 

Ruben

**Comment by FreifunkUFO** [comment]
im asking at ubnt forum to get a airos 5.5 firmware for litebeam, hopefully they are friendly 
https://community.ubnt.com/t5/airMAX-General-Discussion/5-5-AirOS-firmware-for-Litebeam-M5/m-p/1913405#U1913405

**Comment by kpanic23** [comment]
like e.g. http://ftp.ff3l.net/ubiquiti-downgrade/XW.v5.5.10-u2.28005.150723.1358.bin ?

**Comment by FreifunkUFO** [comment]
on ubnt they are not willing to help.
@kpanic23 did you link just a "normal" ubnt firmware (which doesnt support this device) or is it a very special built version of ubnt firmware for my device?

**Comment by FreifunkUFO** [comment]
crazy recovery hints just for bookmarking it: 
https://community.ubnt.com/t5/UniFi-Wireless/HOWTO-Unbrick-your-UniFi-AP/td-p/338684/page/5

**Comment by A-Kasper** [comment]
as long as i know you need to downgrade to 5.5 first after that you are
ready to flash gluon

Am 26.05.2017 17:28 schrieb "FreifunkUFO" <notifications@github.com>:

**Comment by FreifunkUFO** [comment]
please read other comments: atm there is no airos 5.5 version to downgrade!

**Comment by FreifunkUFO** [comment]
now there is a new try, and it was successfully for me :-)
 i had to use ubnt 5.6.4 and was able to flash via webui.

https://patchwork.ozlabs.org/patch/828095/

now i have:

```
root@211-31-litebeam:~# cat /proc/mtd
dev:    size   erasesize  name
mtd0: 00040000 00010000 "u-boot"
mtd1: 00010000 00010000 "u-boot-env"
mtd2: 00760000 00010000 "firmware"
mtd3: 00170000 00010000 "kernel"
mtd4: 005f0000 00010000 "rootfs"
mtd5: 003f0000 00010000 "rootfs_data"
mtd6: 00040000 00010000 "cfg"
mtd7: 00010000 00010000 "EEPROM"

```

**Comment by rotanid** [comment]
@FreifunkUFO it doesn't look like the patch you linked made it into the OpenWrt codebase yet. do you know its status?

@NeoRaider according to the LEDE TOH, the nanobeam m5 xw is supported using the nano loco m5 xw image:
https://lede-project.org/toh/hwdata/ubiquiti/ubiquiti_nanobeam_m5
if this is correct, should we add an alias or something?

**Comment by NeoRaider** [comment]
Current version of the Litebeam support patch: http://patchwork.ozlabs.org/patch/862262/

I guess we can add an alias guarded by a BROKEN check.

**Comment by rotanid** [comment]
@RubenKelevra @FreifunkUFO the NanoBeam M5 is now "supported" (the loco m5 xw image probably would have worked before) but currently marked as BROKEN until there are reports that it is working

**Comment by mweinelt** [comment]
Litebeam M5 is part of OpenWrt 18.06 since 2018/03/02, so if anyone has the device we can work on integrating it.

https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=d0988235dd277b9a832bbc4b2a100ac6e821f577

**Comment by FreifunkUFO** [comment]
this ticket IS about the interest to integrate this device, just like other tickets about hardware.
so, why do you close this ticket?

**Comment by FreifunkUFO** [comment]
@bobcanthelpyou of course yes, i own such a device. see screenshots above. so there is no need to close that ticket, its a "normal" ubnt device

**Comment by bobcanthelpyou** [comment]
@FreifunkUFO could you create merge request please?

**Comment by FreifunkUFO** [comment]
yes, if there is some code i could merge, test or pull-request it.

i can't see any difference to other "new" hardware, where some gluonners start issues (here on github) about supporting it. please reopen that issue-ticket (or: why do you want to see this ticket closed? its a "normal" ubnt- airmax-device)

**Comment by mweinelt** [comment]
Apply this change:

```
diff --git a/targets/ar71xx-generic b/targets/ar71xx-generic
index 96b19d8f..1d0ac529 100644
--- a/targets/ar71xx-generic
+++ b/targets/ar71xx-generic
@@ -266,6 +266,8 @@ device ubiquiti-rocket-m ubnt-rocket-m
 alias ubiquiti-rocket-m2
 alias ubiquiti-rocket-m5

+device ubiquiti-litebeam-m5 ubnt-lbe-m5
+
 device ubiquiti-nanostation-m ubnt-nano-m
 alias ubiquiti-nanostation-m2
 alias ubiquiti-nanostation-m5
```

And build with:

```
make -j$(nproc) GLUON_TARGET=ar71xx-generic DEVICES=ubiquiti-litebeam-m5
```

**Comment by mweinelt** [comment]
@FreifunkUFO Another year has passed. I'm closing this. Please speak up if you intend to test this still.

**Comment by FreifunkUFO** [comment]
Yes, thank you for the Openwrt-Support on Litebeam M5.
So its a now a normal/another ubnt device.

**Comment by mweinelt** [comment]
I'm not sure what that means. Is the LBE-M5 part of the ath79 target? Do you íntend to create a pull request to add Gluon support?

**Comment by suppenkasper0815** [comment]
Hey Guys,

what's about the integration? I found offical support for LiteBeam M5 on openwrt device page. There is an offical sysupgrade and factory image, but of course it won't load via web-ui. 

Now I used SCP to copy newest openwrt image into /tmp and tried it via ssh:

`XW.v6.1.7# fwupdate.real -m fwupdate.bin -d
Found mtd block: /dev/mtd0(u-boot)
Found mtd block: /dev/mtd1(u-boot-env)
Found mtd block: /dev/mtd2(kernel)
Found mtd block: /dev/mtd3(rootfs)
Found mtd block: /dev/mtd4(cfg)
Found mtd block: /dev/mtd5(EEPROM)
Got U-Boot variable: mtdparts = mtdparts=ath-nor0:256k(u-boot),64k(u-boot-env),1024k(kernel),6528k(rootfs),256k(cfg),64k(EEPROM)
Adding U-Boot partition: u-boot 9F000000 00040000
Adding U-Boot partition: u-boot-env 9F040000 00010000
Adding U-Boot partition: kernel 9F050000 00100000
Adding U-Boot partition: rootfs 9F150000 00660000
Adding U-Boot partition: cfg 9F7B0000 00040000
Adding U-Boot partition: EEPROM 9F7F0000 00010000
Calculating flash size:
Adding block: /dev/mtd0("u-boot") - size: 00040000
Adding block: /dev/mtd1("u-boot-env") - size: 00010000
Adding block: /dev/mtd2("kernel") - size: 00100000
Adding block: /dev/mtd3("rootfs") - size: 00660000
Adding block: /dev/mtd4("cfg") - size: 00040000
Adding block: /dev/mtd5("EEPROM") - size: 00010000
Total flash size: 00800000
Flash start: 9F000000
Flash end: 9F800000
Header MAGIC 'UBNT'
Current: XW.ar934x.v6.1.7.32555.180523.1754

New ver: XW.ar934x.v5.6.15.30572.170328.1052
Versions: New(329231) 5.6.15, Required(328971) 5.5.11
FW Part: "u-boot"(1), MAGIC: 'PART', Base: 0x9F000000, DLen: 0x00037CAC, PLen: 0x00040000
FW Part: "kernel"(2), MAGIC: 'PART', Base: 0x9F050000, DLen: 0x000E9789, PLen: 0x00100000
FW Part: "rootfs"(3), MAGIC: 'PART', Base: 0x9F150000, DLen: 0x005E0000, PLen: 0x00660000
FW Part: "script"(4), MAGIC: 'EXEC', Base: 0x00000000, DLen: 0x00006BBD, PLen: 0x00006BBD
Bad Image Structure
Signature check failed
XW.v6.1.7# fwupdate.real -m fwupdate.bin -d -f
Bad Image Structure
Signature check failed
XW.v6.1.7# fwupdate.real -m fwupdate.bin -d -force
Bad Image Structure
Signature check failed
XW.v6.1.7# 
`

I also did this with official Firmware XW.v6.1.7.32555.180523.1754.bin

Here I saw much more ongoing and I guess there was a real flashing because the device runs now 6.1.7.

Unfortunately I have no serial access. I don't know how to open the feedhorn and get it waterproof after opening again. - So as far as I understood in some other discussions we need a beta-image (I registred and accept terms for beta, but couldn't find betaimages at ubiquiti-pages), flash an unsigned old image (offically available on OEM-site) and from there flash the openwrt*.factory image. A direct flash of unsigned images don't work, it stops with error code 43.

I would love if we could get this device to work under openwrt because I wanna use wiregurd with this device and I don't know how to install this on OEM-FW. I guess no chance for it ;) 

Any experience for performance with this device under openwrt?

The device-page in openwrt:

[https://openwrt.org/toh/hwdata/ubiquiti/ubiquiti_litebeam_m5](url)
